zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -167,7 +174,7 @@ class ObjectManagementFacade(
         )
 
         try {
-            logger.trace { "Updating object $objectRequest" }
+            logger.info { "Updating object $objectRequest" }
```

Comment:
I think we shouldn't want to log the whole objectRequest as info here because it might be large, complex and contain sensitive data.

Reply:
I propose to remove the logger line here